api/capi/doc/nntrainer_doc.h

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (C) 2020 Samsung Electronics Co., Ltd. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * @file nntrainer_doc.h
+ * @date 10 July 2020
+ * @brief Tizen C-API Declaration for Tizen SDK
+ * @see	https://github.com/nnstreamer/nntrainer
+ * @author Parichay Kapoor <pk.kapoor@samsung.com>
+ * @bug No known bugs except for NYI items
+ */
+
+#ifndef __TIZEN_MACHINELEARNING_NNTRAINER_DOC_H__
+#define __TIZEN_MACHINELEARNING_NNTRAINER_DOC_H__
+
+/**
+ * @ingroup  CAPI_ML_FRAMEWORK
+ * @defgroup CAPI_ML_NNTRAINER_TRAIN_MODULE Pipeline
```

Comment:
```s/@defgroup CAPI_ML_NNTRAINER_TRAIN_MODULE Pipeline/@defgroup CAPI_ML_NNTRAINER_TRAIN_MODULE Trainer/```?

Reply:
@kparichay @jijoongmoon @zhoonit 
You need to declare group name. e.g., Trainer, NNTrainer, OnDeviceTrainer, ... ... You shouldn't use Pipeline as the name of this group.

Reply:
Updated name of the group to Trainer.